libs/sdk-ui-ext/src/internal/utils/embeddingInsightViewCodeGenerator/insightViewCodeGenerator.ts

```diff
@@ -18,9 +18,10 @@ export const insightViewCodeGenerator = getReactEmbeddingCodeGenerator<IInsightV
         package: "@gooddata/sdk-ui-ext",
     },
     insightToProps: (insightDefinition, ctx) => {
+        const ref = insightId(insightDefinition as IInsight);
```

Comment:
Nitpick: this is not a ref, but id, please rename it, so that it is not misleading.